suricata/update/engine.py

```diff
@@ -163,10 +165,11 @@ def get_version(path):
 
 def test_configuration(suricata_path, suricata_conf=None, rule_filename=None):
     """Test the Suricata configuration with -T."""
+    tempdir = tempfile.TemporaryDirectory()
```

Comment:
This method does not appear to be available on Python 2. `tempfile.mkdtemp` is an option that should work on both.